lib/at_shell/at_shell.c

```diff
@@ -54,6 +54,18 @@ int at_shell(const struct shell *shell, size_t argc, char **argv)
 		at_monitor_pause(&at_shell_monitor);
 		shell_print(shell, "AT command event handler disabled");
 	} else {
+		if (IS_ENABLED(CONFIG_AT_SHELL_UNESCAPE_LF)) {
+			char *c = command;
+
+			/* Replace \n with <CR><LF> so commands which accept multiline input
```

Comment:
should this only be done for CMNG commands?

Reply:
For the current use case, yes. There is also `AT%XPMNG` and `AT%XSUDO` though the presence or absence of EOL characters is not 100% clear in the current docs.